modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductMedia.testcase

```diff
@@ -257,6 +257,38 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-12621.Can view that product image cotains img tag and has the alt attribute "
+	@priority = 5
+	test CanViewAltAttributeOnImgTag {
```

Comment:
Please use a buyer user for this test.
Also, move the test to CommerceMiniCart.testcase or CommerceCheckout.testcase